pkg/client/client.go

```diff
@@ -1674,6 +1674,17 @@ func (c *Client) PodCapacity(ctx context.Context) (int, error) {
 	return int(podCapacityTotal), nil
 }
 
+func (c *Client) GetTelemeterSecretToken(ctx context.Context) (map[string][]byte, error) {
```

Comment:
At this point `GetTelemeterSecretData` sounds like a more descriptive name, what do you think?